src/injector.js

```diff
@@ -0,0 +1,103 @@
+const puma = require('pumascript');
+const http = require('http');
+const fs = require('fs');
+
+
+let errorsLogs = [];
+
+
+function readDependencyJsonFile(dir) {
+    try {
+        return JSON.parse(fs.readFileSync(dir, 'utf8'));
+    } catch (e) {
+        console.log('The file entered does not exist');
+    }
+    return undefined;
+}
+
+
+function readUrl(options, callback) {
+    var req = http.get(options, function (res) {
+        var bodyChunks = [];
+        res.on('data', function (chunk) {
+            bodyChunks.push(chunk);
+        }).on('end', function () {
+            var body = Buffer.concat(bodyChunks);
+            callback(body);
+        })
+    });
+    req.on('error', function (e) {
+        console.log('ERROR: ' + e.message);
+    });
+}
+
+
+
+function createFile(dir, fileName, data) {
+    fs.writeFileSync(dir + '/' + fileName, data);   
+}
+
+//Error object
+function Error(url) {
```

Comment:
`Error` is already a native js object... This should be renamed to something else... If this is supposed to be a custom error then a class extending Error would be best.

```js
class CustomError extends Error {
  constructor(...args, url) {
    super(...args);

    // Maintain proper stack trace
    if (Error.captureStackTrace) {
      Error.captureStackTrace(this, CustomError);
    }

    // Custom information
    this.url = url;
  }
}
```

Reply:
Done.